docs/data/material/components/typography/typography.md

```diff
@@ -27,6 +27,8 @@ Roboto Font has a few easy ways to get started. For more advanced configuration,
 Shown below is a sample link markup used to load the Roboto font from a CDN:
 
 ```html
+<link rel="preconnect" href="https://fonts.googleapis.com" />
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
 <link
   rel="stylesheet"
```

Comment:
Best practice: See the installation instructions https://fonts.google.com/?preview.size=24&sort=popularity